lib/ui/platform_dispatcher.dart

```diff
@@ -2605,3 +2662,52 @@ class SemanticsActionEvent {
     );
   }
 }
+
+/// Signature for [PlatformDispatcher.onPlatformFocusChange].
+typedef ViewFocusChangeCallback = void Function(ViewFocusEvent);
```

Comment:
nit: include an argument name, this enables IDEs to generate better autocomplete suggestions. Otherwise they autogenerate argument names like `arg1`.

```suggestion
typedef ViewFocusChangeCallback = void Function(ViewFocusEvent event);
```
```